tgt_loop.cpp

```diff
@@ -199,6 +226,17 @@ static int loop_init_tgt(struct ublksrv_dev *dev, int type, int argc, char
 		buffered_io = 1;
 	}
 
+	if (bytes > 0) {
+		unsigned long long offset_bytes = offset * (1 << p.basic.physical_bs_shift);
+
```

Comment:
From the helper message of `offset skips first NUM sectors on backing file`, here we should figure out `offset_bytes` via `offset * 512`, which is applied in IO code path too.

Otherwise, this PR looks good.